charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -369,5 +441,5 @@ data:
 {{- end }}
 {{- $unmanagedNamespaceExists := include "unmanaged-resource-exists" (list "rhdh.redhat.com/v1alpha1" "Backstage" .Values.rhdhOperator.subscription.namespace "backstage" .Release.Name .Capabilities.APIVersions ) }}
 {{- if eq $unmanagedNamespaceExists "false" }}
-  {{- include "manage-cr-lifecycle-on-action" (dict "release" .Release "apiGroup" "rhdh.redhat.com" "groupVersion" "v1alpha1" "kind" "Backstage" "kinds" "backstages" "targetNamespace" .Values.rhdhOperator.subscription.namespace "resourceName" "backstage" "isEnabled" .Values.rhdhOperator.enabled "hasCRDInstalled" (.Capabilities.APIVersions.Has "rhdh.redhat.com/v1alpha1/Backstage") "manifest" (include "backstage-manifest" . | b64enc )) }}
+  {{- include "manage-cr-lifecycle-on-action" (dict "release" .Release "apiGroup" "rhdh.redhat.com" "groupVersion" "v1alpha1" "kind" "Backstage" "kinds" "backstages" "targetNamespace" .Values.rhdhOperator.subscription.namespace "resourceName" "backstage" "isEnabled" .Values.rhdhOperator.enabled "hasCRDInstalled" (.Capabilities.APIVersions.Has "rhdh.redhat.com/v1alpha1/Backstage") "manifest" (include "backstage-manifest" $context | b64enc)) }}
```

Comment:
Why do you need to pass `$context` here? I don't see the `manage-cr-lifecycle-on-action` function being modified to include logic to manage the `"cert" $cert` variable. I'm not sure I understand what you're trying to achieve here.

Reply:
I need to have the cert inside the define section, so I need to switch to context to be able to do the if on cert inside the backstage object.

Reply:
I'm not sure I understand what you're saying here: Can you show me where is `cert` being referenced in the `manage-cr-lifecycle-on-action` function?
[charts/orchestrator/templates/_cr_lifecycle_helper.tpl](https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator/templates/_cr_lifecycle_helper.tpl)